MAINTENANCE_STANDARDS.md

```diff
@@ -0,0 +1,24 @@
+This document outlines a standardized policy for when Dependabot will cease guaranteeing support with package managers, and for when it begin supporting a new version. "Cease guaranteeing support" is intentionally distinct from "deprecating support", because just because something's not supported, doesn't necessarily mean that we need to "rip out support immediately". Instead, it means more like, "if supporting this becomes a problem, we won't fix it".
```

Comment:
```suggestion
This document outlines a standardized policy for when Dependabot will cease guaranteeing support for old versions of package managers and languages, and for when it will add support for new versions. "Cease guaranteeing support" is intentionally distinct from "deprecating support", because just because something's not supported, doesn't necessarily mean that we need to "rip out support immediately". Instead, it means more like, "if supporting this becomes a problem, we won't fix it".
```